### DISCUSSION TITLE
Is it possible to add content on the somewhat unused right side of edit page?

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#981] [Q&A] [answered]
In the add/edit page of Twill for one of my modules I want to add some extra tips on content editing.   It would be great to use some of the empty space below the right side status/update box.    Stacking some extra content there is what WordPress does in it's classic interface (sorry if that is a dirty word here) and I thought that in this respect it would be useful to replicate something similar.

Is there any way of adding this via view files or something similar ?

![image](https://user-images.githubusercontent.com/3810108/122241979-f7075f00-ce90-11eb-8237-2e34bcd69209.png)

**Comment by pboivin** [comment, marked as answer]
Hey @arkid,

This is not yet well documented but there are a few Blade sections you can use to inject content into your CMS pages. The one you are looking for is `sideFieldsets` :

**resources/views/admin/mymodule/form.blade.php**

```
@extends('twill::layouts.form')

@section('contentFields')
    ...
@stop

@section('sideFieldsets')
    <div>
        HELLO WORLD!
    </div>
@stop
```

To get an overview of what's available, you can search for `@yield` in the Twill source code (under views/layouts/).

Hope this helps!

**Reply by adam-jones-net** [reply]
Fantastic thanks for sharing @pboi20  !
